### ISSUE TITLE
Post Download action for more than zip files

### ISSUE THREAD

**Issue by NerdFlix** [zach-morris/plugin.program.iagl#3]
Hi zach. Is it possible to include a post download action for more than zip files? I think Kodi has built in support now(I'm not sure it's been a long time since I've dealt with Kodi/python), I have custom lists with rar and other compressed types, maybe a new PDA that decompress more types of files and rename the resulting rom according to XML(dat file) would be helpful, decompressing splitted files would be a plus.

Kodi forum about it(I think you had the same question as me): https://forum.kodi.tv/showthread.php?tid=323321

Thanks.

**Comment by zach-morris** [comment]
I am planning on adding support for [vfs.libarchive](https://github.com/notspiff/vfs.libarchive), which I think will be available as a standard addon for all platforms for Kodi v18.  The only reason it isn't added yet is because it isn't available for OSX (which is the platform I develop on).  This is coming too though.

**Comment by NerdFlix** [comment]
Good news then, thanks

**Comment by zach-morris** [comment]
Included in 2.1.0.  vfs.libarchive adds support for:
```
['.7z','.tar.gz','.tar.bz2','.tar.xz','.zip','.rar','.tgz','.tbz2','.gz','.bz2','.xz']
```
